readme.md

```diff
@@ -496,6 +496,7 @@
 - [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Automated software testing technique that involves feeding pseudo-randomly generated input data.
 - [Embedded and IoT Security](https://github.com/fkie-cad/awesome-embedded-and-iot-security#readme)
 - [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Regulation on data protection and privacy for all individuals within EU.
+- [Reversing](https://github.com/ReversingID/awesome-reversing#readme) - Reverse Engineering tools, techniques, and references for various topics.
```

Comment:
> `Has consistent formatting and proper spelling/grammar.`
    ✅  `The link and description are separated by a dash.`
    ❌  `The description starts with an uppercase character and ends with a period.`

In all of your specialization pages there are instances of descriptions missing the uppercase at the beginning and/or period at the end.